utilities/db-migration/migration_test/add_test_values/add_db_values.go

```diff
@@ -68,8 +68,8 @@ var (
 		Health:                          "Healthy",
 		Sync_Status:                     "Synced",
 		ReconciledState:                 "test-reconcile",
-		SyncError:                       "test-sync-error",
 		OperationState:                  []byte("operation_state"),
+		Conditions:                      []byte("conditions"),
```

Comment:
We can also a check of this value in `utilities/db-migration/migration_test/verify_test_values/verify_db_values_test.go`